### PR TITLE
The fusion generators no longer break down.

### PR DESCRIPTION
**Pull request by BeloneX**
No compromises for terrible game design.

## Changelog
:cl:
tweak: The fusion generators no longer randomly break down.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -20. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Walarks** [comment]
Why is this needed considering ship technicians don't have much else to do?

**Comment by BeloneX** [comment]
> Why is this needed considering ship technicians don't have much else to do?

Because it is a chore, and it is not gameplay.

File: code/modules/power/fusion_engine.dm [hunk 1]
```diff
@@ -18,7 +18,7 @@
 	var/power_gen_percent = 0 //80,000W at full capacity
 	var/buildstate = FUSION_ENGINE_NO_DAMAGE //What state of building it are we on, 0-3, 1 is "broken", the default
 	var/is_on = FALSE  //Is this damn thing on or what?
-	var/fail_rate = 5 //% chance of failure each fail_tick check
+	var/fail_rate = 0 //% chance of failure each fail_tick check
 	var/cur_tick = 0 //Tick updater
```
Comment (TiviPlus) [inline code/modules/power/fusion_engine.dm:23]: just remove check_failure and affiliated code

**Review by TiviPlus — CHANGES_REQUESTED** [review]

File: code/modules/power/fusion_engine.dm (hunk 1 above)
Comment (BeloneX) [reply code/modules/power/fusion_engine.dm]: Done.

**Comment by BeloneX** [comment]
@TiviPlus Done.

**Comment by BeloneX** [comment]
God I hate linters.

**Comment by BeloneX** [comment]
image
Nevermind, I believe this is outside of my control

**Review by TiviPlus — APPROVED** [review]

**Review by SplinterGP — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-22). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**